### PR TITLE
Add ciral bm25 baselines and experiment doc

### PR DESCRIPTION
**Pull request by Mofetoluwa**
This cl includes:
- CIRAL's bm25 baselines for all 4 languages: Hausa, Somali, Swahili, Yoruba.

Comments/Issue:
- Just `recall@1000` is evaluated currently, as `ndcg@20` and `recall@100` for Swahili and Hausa are 0.

### CODE REVIEW

**Review by lintool — COMMENTED** [review]
hi @Mofetoluwa - the participant _either_ downloads the pre-built indexes or indexes from scratch, right? Doesn't need to do _both_?

Perhaps this can be reflected in the docs?

**Comment by Mofetoluwa** [comment]
Hi Jimmy, yeah that's true, I was kinda following the format for the current Pyserini regressions documents.

I'll remove the prebuilt indexes downloads, or do I just leave a comment about it in the doc?

File: docs/experiments-ciral-v1.0.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+python -m pyserini.eval.trec_eval -c -m recall.1000 tools/topics-and-qrels/qrels.ciral-v1.0-sw-train.tsv runs/run.ciral-v1.0-sw.bm25.topics.ciral-v1.0-sw.train.txt
+python -m pyserini.eval.trec_eval -c -m recall.1000 tools/topics-and-qrels/qrels.ciral-v1.0-yo-train.tsv runs/run.ciral-v1.0-yo.bm25.topics.ciral-v1.0-yo.train.txt
+```
+
+## Effectiveness
+
+### Yoruba
+
+With the above commands, you should be able to reproduce the following results:
+| **recall@1000**                                                                   | **BM2 (default)**|
```
Comment (lintool) [inline docs/experiments-ciral-v1.0.md:109]: Should be BM25.
Why not evaluate also with MRR or nDCG?

Comment (Mofetoluwa) [reply docs/experiments-ciral-v1.0.md:109]: Hi Jimmy,
Do you think we should hold on about this based on the results?

File: docs/experiments-ciral-v1.0.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+cp -r ciral/*/*/* $PYSERINI_PATH/tools/topics-and-qrels/
+```
+
+
+
+## Retrieval: Dev Topics
+
+```bash
+python -m pyserini.search.lucene --index ciral-v1.0-ha \
+    --topics tools/topics-and-qrels/topics.ciral-v1.0-ha-train \
```
Comment (crystina-z) [inline docs/experiments-ciral-v1.0.md:71]: we hvn't add the topics to the tools/ yet?

Comment (crystina-z) [reply docs/experiments-ciral-v1.0.md:71]: also could we name it `dev` instead of `train`? that'd be more accurate

Comment (lintool) [reply docs/experiments-ciral-v1.0.md:71]: our just call it "sample"?

**Review by crystina-z — APPROVED** [review]

**Pull request closed without merging**